x-pack/test/accessibility/apps/snapshot_and_restore.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'settings', 'header', 'snapshotRestore']);
+  const a11y = getService('a11y');
+  const es = getService('es');
+
+  describe('Stack management- snapshot restore a11y tests', () => {
+    before(async () => {
+      await PageObjects.settings.navigateTo();
+      await PageObjects.settings.clickSnapshotRestore();
+    });
+
+    describe('empty state', async () => {
+      it('empty snapshots table', async () => {
+        await a11y.testAppSnapshot();
+      });
+
+      it('empty repositories table', async () => {
+        await PageObjects.snapshotRestore.navToRepositories();
+        await a11y.testAppSnapshot();
+      });
+
+      it('empty policies table', async () => {
+        await PageObjects.snapshotRestore.navToPolicies();
+        await a11y.testAppSnapshot();
+      });
+
+      it('empty restored snapshots status table', async () => {
+        await PageObjects.snapshotRestore.navToRestoreStatus();
+        await a11y.testAppSnapshot();
+      });
+    });
+
+    describe('table views with data', async () => {
+      const snapshotName = `testsnapshot${Date.now().toString()}`;
+      before(async () => {
+        await es.snapshot.createRepository({
+          name: 'testrepo',
+          verify: true,
+          type: 'fs',
+          repository: {
+            type: 'fs',
+            settings: {
+              location: 'temp',
+            },
+          },
+          settings: {
+            location: 'temp',
+          },
+        });
+
+        await es.snapshot.create({
+          repository: 'testrepo',
+          snapshot: snapshotName,
+          wait_for_completion: true,
+        });
+
+        await PageObjects.settings.navigateTo();
+        await PageObjects.settings.clickSnapshotRestore();
+      });
+      it('snapshots table with data', async () => {
+        await a11y.testAppSnapshot();
+      });
+      it('repository table with data', async () => {
+        await PageObjects.snapshotRestore.navToRepositories();
+        await a11y.testAppSnapshot();
+      });
+
+      describe('create policy wizard', async () => {
```

Comment:
I'm not sure it makes sense to nest 'create policy wizard' tests within 'table views with data' tests. I see the Wizard View as more of a sibling to the Table View, so I'd suggest mirroring this in the test structure by making 'create policy wizard' tests and 'table views with data' tests siblings to one another.

Reply:
OK that makes sense. I nested it based on how you get to it in the flow instead of functionality. I will adjust that.